src/renderer/protocol-handler/app-handlers.tsx

```diff
@@ -30,14 +31,25 @@ import { entitySettingsURL } from "../components/+entity-settings";
 import { catalogEntityRegistry } from "../api/catalog-entity-registry";
 import { ClusterStore } from "../../common/cluster-store";
 import { EXTENSION_NAME_MATCH, EXTENSION_PUBLISHER_MATCH, LensProtocolRouter } from "../../common/protocol-handler";
+import { Notifications } from "../components/notifications";
 
 export function bindProtocolAddRouteHandlers() {
   LensProtocolRouterRenderer
     .getInstance()
     .addInternalHandler("/preferences", ({ search: { highlight }}) => {
       navigate(preferencesURL({ fragment: highlight }));
     })
-    .addInternalHandler("/", () => {
+    .addInternalHandler("/", ({ tail }) => {
+      if (tail) {
+        Notifications.info((
+          <p>
+            Missing handler for <code>lens://app/{tail}</code>. Are you on the latest version?
```

Comment:
Handlers might be removed in a "latest" version too, right? "correct" might be better than "latest"